Servers/Azure/ARC/instructions.md

```diff
@@ -0,0 +1,49 @@
+# Accessing Azure Account Server From Out of Office
```

Comment:
if you change the filename to README.md, then GitHub will automatically render and display it whenever someone visits the Servers/Azure/ARC page
(like we did here: https://github.com/ReactionMechanismGenerator/RMG-Py/tree/main/arkane)